server/src/main/java/io/crate/metadata/doc/DocIndexMetadata.java

```diff
@@ -547,6 +547,7 @@ private void internalExtractColumnDefinitions(@Nullable ColumnIdent parent,
                         IndexReference.Builder builder = getOrCreateIndexBuilder(newIdent);
                         builder.indexType(columnIndexType)
                             .position(position)
+                            .oid(oid)
```

Comment:
Was bug on feature branch, spot by `IndexReferenceTest.test_mapping_generation_ft_col_has_source_columns` after supporting OID in tables created by `SQLExecutor.addTable`